frontend/src/app/commonComponents/Dropdown.tsx

```diff
@@ -14,7 +14,7 @@ interface Props {
   options: Option[];
   label?: string;
   name?: string;
-  onChange: (e: React.FormEvent<HTMLSelectElement>) => void;
+  onChange: (e: React.ChangeEvent<HTMLSelectElement>) => void;
```

Comment:
This was just fixing the type of the event handler, it's also in the big PR but I figured I should pull it over.